src/lib/containers/IDUReportSubmissionInfo.tsx

```diff
@@ -26,7 +26,7 @@ export const SubmissionInfoCard: React.FunctionComponent<
           <strong>
             {' '}
             Intended Data Use Statement &#40;accepted on{' '}
-            {formatDate(moment(info.modifiedOn), 'M/D/YYYY')}&#41;:
+            {formatDate(dayjs(info.modifiedOn, 'M/D/YYYY'))}&#41;:
```

Comment:
Wouldn't the default pattern in `formatDate` end up overriding the one specified here?
```suggestion
            {formatDate(dayjs(info.modifiedOn), 'M/D/YYYY')}&#41;:
```